### DISCUSSION TITLE
Does it make sense to have two installs of SD.Next on the same machine ? One old-school and one for diffuser models ?

### DISCUSSION THREAD

**Discussion by swamp-cabbage** [vladmandic/sdnext#1625] [General]
I'm not sure what the landscape is going to look like with new models and LoRAs, but I was thinking about a separate --diffusers install for SDXL with it's own file hierarchy.

Does that make sense ?

**Comment by vladmandic** [comment]
not at all, just use `--backend diffusers` and `--backend original`

**Comment by swamp-cabbage** [comment]
Well that was quick and decisive.

Thanks Bro !
